packages/reports/src/Components/indicative-cell.tsx

```diff
@@ -1,11 +1,28 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Icon, Money, DesktopWrapper, ContractCard } from '@deriv/components';
 import { getCardLabels } from '_common/contract';
 import { connect } from 'Stores/connect';
+import { TRootStore } from 'Stores/index';
 
-const IndicativeCell = ({ amount, currency, contract_info, is_footer, onClickSell, is_sell_requested }) => {
-    const [movement, setMovement] = React.useState(null);
+type TIndicativeCell = {
+    amount: number;
+    contract_info: object;
+    currency: string;
+    status: string;
+    is_footer: boolean;
+    is_sell_requested: () => void;
```

Comment:
Are you sure it's a function? In my opinion it's boolean or function which returns boolean: "() => boolean"

Reply:
I don't think it's a function, should be `boolean` as well.

Reply:
Normally, we use camelCase for functions and snake_case for normal variables

Reply:
Will take a look at this thank you for pointing it out 🙏🏼